indico/web/client/js/legacy/libs/timetable/Draw.js

```diff
@@ -180,11 +156,12 @@ type(
         this._getTitle()
       );
 
-      this.titleWrapper = Html.div({}, this.titleDiv);
+      this.headerBlock = Html.div({className: 'timetableBlockWrapper'}, this.titleDiv);
+      this.footerBlock = Html.div({className: 'timetableBlockWrapper', style: { marginTop: 'auto' }});
```

Comment:
even though this is legacy code and will probably be removed at some point I would still recommend to be consistent and maybe remove the spaces inside in the object literal

Reply:
Agree!